pkg/kv/cosmosdb/main_test.go

```diff
@@ -60,6 +60,9 @@ func TestCosmosDB(t *testing.T) {
 }
 
 func TestMain(m *testing.M) {
+	// SKIP CosmoDB tests until we find a bettwe way to test with the curret emulator which fail our tests
```

Comment:
@nopcoder Please add a link to the issue that suppose to make this test work again

Reply:
it is in the description